client/settings/express-checkout-settings/payment-request-button-preview.js

```diff
@@ -67,6 +69,42 @@ const buttonSizeToPxMap = {
 	large: 56,
 };
 
+const WooPayButtonPreview = ( { size, buttonType, theme } ) => (
+	<WoopayExpressCheckoutButton
+		isPreview={ true }
+		buttonSettings={ {
+			type: buttonType,
+			text: 'Buy',
+			theme: theme,
+			height: `${
+				buttonSizeToPxMap[ size ] || buttonSizeToPxMap.medium
+			}px`,
+			size,
+		} }
+	/>
+);
+
+const ButtonPreviewWrapper = ( { theme, children } ) => (
+	<>
+		<div className="payment-method-settings__preview" data-theme={ theme }>
+			{ children }
+		</div>
+		<BrowserHelpText />
```

Comment:
If we decide to show both button previews when available we need to add the Safari notice only when using Chrome, and never show the notice in Safari, but only when ECE is enabled.

Reply:
Also related to the inline notice, and now that we're displaying all buttons when available, should we remove the `BrowserHelpText` now that we have this new notice in place? We may display it just on Chrome, however, we may want to hide it when the inline notice is being shown.

**On Brave/Chrome with an insecure connection**

<img width="662" alt="Screenshot 2024-07-08 at 18 27 12" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/42baeba7-ab6f-404d-91fa-a21423b09fe6">

**On Chrome**

<img width="659" alt="Screenshot 2024-07-08 at 18 21 03" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/97a353a3-4435-4747-a43d-846639ad64d5">

**On Safari**

<img width="666" alt="Screenshot 2024-07-08 at 18 24 30" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/ed27c741-06ac-4e22-9d08-5042a879cf26">

Reply:
> should we remove the BrowserHelpText now that we have this new notice in place?

Yes, this helper text seems redundant with the notice. I think it could be removed since the notice will provide enough info.

Reply:
Removed the help text in 4a7cdbf92876153d21a706716f489e836cf03f47.

Reply:
Gotcha. I missed that you meant the https notice is redundant to the browser notice. Looks good.